config/global.ini.php

```diff
@@ -456,6 +456,10 @@
 ; the page will automatically refresh every 5 minutes. Set to 0 to disable automatic refresh
 multisites_refresh_after_seconds = 300
 
+; by default, update notifications for a new version of Piwik is shown to every user. Set to 1 if only
+; the superuser should see the update notification.
```

Comment:
Description could be improved: Better would be `by default, an update notification for a new version of Piwik is shown...` or `by default, update notifications for new versions of Piwik are shown...`, also I guess it would affect `all superusers`, as there might be more than one.

Reply:
Good point, this was confusing. Changed it a bit - should be more clear to the user now.